app/views/admin/admin_users/_user_creations_summary.html.erb

```diff
@@ -0,0 +1,18 @@
+<% unless @works.size.zero? %>
```

Comment:
Is there a reason why `size.zero?` is used over `empty? `?

Reply:
Nope, that's just what it was originally, so I didn't change it.